content/docs/dvclive/api-reference/live/log_param.md

```diff
@@ -0,0 +1,60 @@
+# Live.log_param()
+
+Logs the given parameter `val` associating it with the given `name`.
+
+```py
+ def log_param(name: str, val: ParamLike):
+```
+
+Also see `Live.log_params()`.
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+
+live.log_param("lr", 1e-3)
+live.log_param("n_iter", 50)
+```
+
+## Description
+
+On each `live.log_param(name, val)` call, DVCLive will write the `name`
+parameter to `{Live.dir}/params.yaml` with the corresponding `val`:
+
+```dvc
+$ cat dvclive/params.yaml
+lr: 1e-3
+n_iter: 50
+```
+
+<admon type="tip">
+
+The parameters file `{Live.dir}/params.yaml` is usable by `dvc params`,
+`dvc metrics` and `dvc exp show`/`dvc exp diff`.
+
+</admon>
```

Comment:
It's not that clear how to use it with any of these commands, or why we would suggest both `dvc params` and `dvc metrics`. What do you think about leaving this out until we add better support for tracking it in DVC?